src/AzureAuth/CommandMain.cs

```diff
@@ -120,6 +126,12 @@ public CommandMain(CommandExecuteEventData eventData, ITelemetryService telemetr
         [Option(PromptHintOption, "The prompt hint text for WAM prompts and web mode.", CommandOptionType.SingleValue)]
         public string PromptHint { get; set; }
 
+        /// <summary>
+        /// Gets or sets global Timeout.
+        /// </summary>
+        [Option(TimeoutOption, "Number of seconds for which the CLI should run", CommandOptionType.SingleValue)]

```

Comment:
I know we have some back and forth on whether to use seconds or minutes. Seconds is common, but here almost never make sense, and devs have an easier time thinking in minutes they wait. We are using a double - to allow for sub-minute granularity, and I thought by using doubles, @reillysiemens had agreed that interpreting as minutes was okay here.

I thought of this again because of this suggestion where I want to say "default is 10 minutes" but I don't want to say 10 minutes if the actual value to make that happen is actually `--timeout 600`. You could do "default is 600 seconds" But then I 'd want to add `(10 minutes)`. I'm very against making people do math for no reason :). 
```suggestion
        [Option(TimeoutOption, "The number of seconds the CLI is allowed to wait before timing out.\nDefault: 10 minutes.", CommandOptionType.SingleValue)]

```

Reply:
@kyle-rader would it be better in that case to have the option interpret the number as minutes directly?